misc/convenientDiscussions-generateBasicConfig.js

```diff
@@ -151,13 +156,11 @@ mw.loader.using([
     ]
   );
 
-  const signatureResp = await new mw.Api().post({
-    action: 'parse',
-    page: 'MediaWiki:Signature',
-    prop: ['text'],
-    formatversion: 2,
-  });
-  const parsedSignature = signatureResp?.parse?.text;
+  const signatureMessage = (await api.getMessages('Signature', {
+    amlang: mw.config.get('wgContentLanguage'),
+    amincludelocal: 1
+  })).Signature;
```

Comment:
![image](https://user-images.githubusercontent.com/33615628/114114834-5887dc00-98ea-11eb-940f-d1041a131828.png)
`mw.Api#getMessages` returns data in this format, as I can see?

Reply:
That's the full output of [API:Allmessages](https://www.mediawiki.org/wiki/API:Allmessages). The shortcut function [`getMessages`](https://doc.wikimedia.org/mediawiki-core/master/js/source/messages.html#mw-Api-plugin-messages-method-getMessages) should return `{ Signature: '<message text>' }`

Reply:
Ah right, sorry.